plugins/scaffolder-backend/src/scaffolder/actions/builtin/stress/waitWorkflow.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTemplateAction } from '../../createTemplateAction';
+
+export function waitWorkflow() {
```

Comment:
I'm not sure that we want to expose the stress template and this action to users? It does not provide any usefulness more than testing the wait logic right?

It would be great if we could add some unit tests for the TaskWorker to ensure that this new behaviour work as expected :)

Reply:
Tests Added :)